cmd/is-ready.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright © 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"syscall"
+
+	"github.com/JAORMX/selinuxd/pkg/daemon"
+	"github.com/spf13/cobra"
+)
+
+// isreadyCmd represents the is-ready command
+var isreadyCmd = &cobra.Command{
+	Use:   "is-ready [policy]",
+	Short: "probe ready endpoint",
+	Long:  `Tells whether selinuxd is ready or not.`,
+	Run:   isreadyCmdFunc,
+}
+
+// nolint:gochecknoinits
+func init() {
+	rootCmd.AddCommand(isreadyCmd)
+	defineIsReadyFlags(isreadyCmd)
+}
+
+func defineIsReadyFlags(rootCmd *cobra.Command) {
+	rootCmd.Flags().String("socket-path", daemon.DefaultUnixSockAddr, "The path to the socket to listen at")
```

Comment:
Nit: the description is wrong, it should say something like 'the path where selinuxd socket is listening at"